.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags: 
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  docker_job:
+    name: Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - id: theVersion
+        run: echo "::set-output name=theVersion::$(echo ${GITHUB_REF##*/})"
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Publish Docker image
+        run: |
+          make build
+          make docker-push
+        env:
+          VERSION: ${{ steps.theVersion.outputs.theVersion }}
+      - name: Upload Helm artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: "helm"
+          path: "opa-${{ steps.theVersion.outputs.theVersion }}.tgz"
+
+  helm_job:
```

Comment:
It would be helpful if could add some comments as to what this job is supposed to do.

Reply:
Where should I put those comments ? What sort of comments, plz clarify more what and where you would like to see the comments.